src/current/v23.1/cluster-setup-troubleshooting.md

```diff
@@ -441,6 +441,16 @@ CockroachDB's built-in disk stall detection works as follows:
 
 - During [node liveness heartbeats](#node-liveness-issues), the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) writes to disk as part of the node liveness heartbeat process.
 
+#### Disk usage is different across nodes in the cluster
+
+This is expected behavior.
+
+CockroachDB uses [load-based replica rebalancing]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing) to spread [replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) across the nodes in a cluster.
+
+The rebalancing criteria for load-based replica rebalancing do not include 'disk usage per node'. Not all [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) are the same size. Larger and smaller ranges balance each other out on average, so disk usage differences across nodes should be relatively small.
```

Comment:
Do you think we should also make mention of utilization? We strongly don't recommend running with different disk sizes, however the remaining capacity is what users care about -- which is more tightly linked with utilization, than usage.

Reply:
Ah ok I think I was using them as synonyms but after reading your comment it sounds like:

- usage: "total GiB (or whatever) of storage on disk per node"
- utilization: "percentage of disk capacity used per node"

Is that right?  If so then the latter is what we are actually talking about here, since different disks having different "percent fullnesses" is what was driving the customers to file issues, yes?

I have updated the PR to change "usage" to "utilization", I hope I've used the term correctly, please take a look and let me know if not

Reply:
> Is that right? If so then the latter is what we are actually talking about here, since different disks having different "percent fullnesses" is what was driving the customers to file issues, yes?

Yup

> I have updated the PR to change "usage" to "utilization", I hope I've used the term correctly, please take a look and let me know if not

Will do.